### ISSUE TITLE
bicycles are still allowed on the main road even if bicycle lane was added (trac #2350)

### ISSUE THREAD

**Issue by behrisch** [eclipse/sumo#2350]
compound type 'cycleway.track|highway.secondary' imported from OSM
Some of the imported edges have correct permissions and others with the same compound type do not.

See sumo2016 tutorial 'all_modes' (select lanes which allow bicycle)

Migrated from http://sumo.dlr.de/ticket/2350
```json
{
    "status": "new", 
    "changetime": "2016-10-14T10:39:37Z", 
    "description": "compound type 'cycleway.track|highway.secondary' imported from OSM\nSome of the imported edges have correct permissions and others with the same compound type do not.\n\nSee sumo2016 tutorial 'all_modes' (select lanes which allow bicycle)", 
    "reporter": "namdre", 
    "cc": "", 
    "resolution": "", 
    "_ts": "1476441577749276", 
    "component": "net building (NETGEN, NETCONVERT)", 
    "summary": "bicycles are still allowed on the main road even if bicycle lane was added", 
    "priority": "major", 
    "keywords": "", 
    "time": "2016-05-23T10:42:30Z", 
    "milestone": "1.0.0", 
    "owner": "", 
    "type": "defect"
}
```

**Comment by namdre** [comment]
recheck whether this should be replaced with double-connections

**Comment by namdre** [comment]
fix depends on #7550
